test/tools/Modules/WebListener/WebListener.psm1

```diff
@@ -158,7 +158,7 @@ function Get-WebListenerUrl {
             return $null
         }
         $Uri = [System.UriBuilder]::new()
-        $Uri.Host = 'localhost'
+        $Uri.Host = '127.0.0.1'
```

Comment:
Could you please add comment and link on CoreFX Issue?

Reply:
done.